code/modules/clothing/spacesuits/rig/modules/modules.dm

```diff
@@ -347,3 +348,14 @@
 		name = "[charge.display_name] ([charge.charges]C) - Change"
 		return 1
 	return 0
+
+/mob/living/carbon/human/ClickOn(atom/A, params)
```

Comment:
uhm... this is not good. First this is not where this proc should be placed. S

It would be better if you move it to human_defense.dm. Also maybe move this code into attack_generic

Reply:
What poze said. Using the click-API is skipping a few steps.

Reply:
What's the difference between clickon and attack_generic

Reply:
`attack_generic()` is for non-human mobs only. I told Kaed to override ClickOn so that the code would always work, even for objects that improperly override attack*.

Reply:
Also this acts on the item, not the mob. Why does it need to move to humancode?

Reply:
\> Writing bad code because the rest of the code is already written badly.

fine.

And no, this fits here, Poze.